src/components/App/App.tsx

```diff
@@ -33,57 +33,78 @@ import GlobalStyle from './GlobalStyle';
 import muiTheme from './muiTheme';
 import config from '../../config';
 
-const StyledDiv = styled.div`
+interface AppStylesProps {
+  open?: boolean;
+  location: {
+    pathname?: string;
+  };
+}
+
+interface Back2TopStylesProps {
+  open?: boolean;
+  location: {
+    pathname: string;
+  };
+}
+
+interface AppProps {
+  location: { [key: string]: string }
+  strings: { [key: string]: string }
```

Comment:
Can we use one shared type for strings so if we figure out how to type it better later we only have one spot to update?

Reply:
can use `GlobalStrings` in `types/common` directory, right?

Reply:
Yeah if that's an alias

Reply:
hmmm... I think will be facing the second option mentioned here: https://github.com/odota/web/pull/3124#discussion_r1398334357

Reply:
OK, in the meantime can you point all the string references to one place instead of duplicating { [key: string]: string }?

Reply:
Actually it's probably best to use Record<string, string> which is a builtin type

Reply:
I just updated the PR with the builtin type as you suggested. I can go ahead and update other instances of `{ [key: string]: string }` in a separate PR if you approve.

thanks and sorry for late reply :)